packages/venia-ui/lib/components/Header/accountTrigger.js

```diff
@@ -47,11 +63,16 @@ const AccountTrigger = props => {
                 </button>
             </div>
             <AccountMenu
-                handleSignOut={handleSignOut}
-                isOpen={accountMenuIsOpen}
-                isUserSignedIn={isUserSignedIn}
                 ref={accountMenuRef}
+                view={view}
+                username={username}
+                isOpen={accountMenuIsOpen}
+                handleSignOut={handleSignOut}
+                handleForgotPassword={handleForgotPassword}
+                handleCreateAccount={handleCreateAccount}
+                updateUsername={setUsername}
             />
+            <FormErrors errors={formErrors} />
```

Comment:
I'm not sure the sign out error is considered a form error, but I ran in to similar issues in #2588 where some mutation/query returns an error that we want to parse and render. Since signing out is not done in a form I would ask that you remove the use of it here. I can then make a note to pop a toast in #2588 since I have the `deriveErrorMessage` utility function.

Reply:
That sounds fine to me. Signout certainly feels like a toast error than a form error.